#### Case Study: An Alien Caused An Epidemic

During a specific epidemic dating back to mid 1900s, it was
estimated that more than 8,320,000 people had died and 25 million
had been inflicted with a particular infectious disease caused
by an unknown causative agent.

Several reasons were made to account for its devastating damages
including wrath of god, alien invasion, and a pathogen. Suppose
you were a researcher back in mid 1900s with all of the present-day
utilities. Using molecular techniques, a sequence of the "_alien_" was
obtained and found out. How would you characterize this organism?

#### DO NOT TOUCH STARTING FROM HERE!

In [1]:
# open the data set
with open("../data/alien_genome.txt") as seq_:
    seq = seq_.read()

In [2]:
print(seq[-100:]) # last 100 nt of the data set

TCGAACGAAAATAAATTGCTCGTTCGCCATTTCTTTACTGCGGTGATGCTCACATCTGTACTGTGAGTTCCGATAGTAGAGGTTTATACCATCGCAAAA



In [3]:
print(f"{len(seq):,}") # number of nt of the data set

1,108,251


#### Fundamental Properties of Data Set

**Genome Length**: 1,108,251 ($1.108 \cdot 10^6$) nt

**Organism**: Not a Fucking Alien, or is it?

#### Objectives

1. Obtain a transcript of the genome sequence.
2. Obtain the translate of the transcript.
3. Create a phylogenetic tree of the genome sequence.
4. Identify the causative agent.

#### START YOUR WORKSHOP HERE!

In [4]:
# this is a python cell
# start here